vibes/vibed-ui/examples/demo_controls.cpp

```
@@ -0,0 +1,85 @@
+    tree.setRoot(root);
+
+    ui::Theme theme;
+    (void)theme;
+
+    bool shouldExit = false;
+    for (int frame = 0; frame < 120 && !shouldExit; ++frame) {
+        if (platform::activePlatform() != nullptr) {
+            platform::activePlatform()->pumpEvents();
+        }
+
+        core::Event event;
```

Clang (14.0.6)

[warning] variable 'event' is not initialized

(cppcoreguidelines-init-variables)